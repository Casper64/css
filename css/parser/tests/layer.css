
@layer test other;

@layer test, , other;

@layer test {
    .test {
        color: green;
    }
