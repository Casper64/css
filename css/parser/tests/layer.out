css/parser/tests/layer.css:2:13: warning: expecting a comma separated list. Missing "," before "other"
    1 | 
    2 | @layer test other;
      |             ~~~~~
    3 | 
    4 | @layer test, , other;
css/parser/tests/layer.css:4:14: warning: expecting a name, ";" or "{" not ","
    2 | @layer test other;
    3 | 
    4 | @layer test, , other;
      |              ^
    5 | 
    6 | @layer test {
css/parser/tests/layer.css:10:1: error: expecting "}". Did you forget to close the previous block?
    8 |         color: green;
    9 |     }